jest.config.ts

```diff
@@ -75,15 +75,16 @@ const projects = [
             LOCAL_HOST_BINDING: '0.0.0.0'
         },
         runner: 'jest-serial-runner'
-    }
-    // TODO uncomment once Python is ready
-    // {
-    //     ...commonConfig,
-    //     displayName: 'Python',
-    //     globals: {
-    //         JEST_PROJECT_SDK_TO_TEST: 'Python',
-    //     }
-    // },
+    },
+    {
+        ...commonConfig,
+        displayName: 'Python',
+        globals: {
+            JEST_PROJECT_SDK_TO_TEST: 'Python',
+            LOCAL_HOST_BINDING: '0.0.0.0',
+        },
+        runner: 'jest-serial-runner'
```

Comment:
Do we need the serial runner for Python? It seems to work for me locally without, and I'm not aware of any concurrency issues (yet).

Reply:
OK, it seems to actually run *faster* with the serial runner enabled. So maybe we should leave it in place.

Reply:
good point.  you are right in that it probably doesn't.  I think i put it there to help with dev/debugging

Reply:
i'll remove that

Reply:
lol.  i'll put it back